OWML.ModHelper/Logging/ModSocketOutput.cs

```diff
@@ -46,26 +43,49 @@ public override void WriteLine(string s)
             {
                 type = MessageType.Message;
             }
-            WriteLine(type, s);
+            var senderName = Manifest.Name;
+            var senderFile = (new System.Diagnostics.StackTrace()).GetFrame(1).GetMethod().DeclaringType.Name;
```

Comment:
You're doing this in two places. Write a method for it, and make sure it still works in case this throws an error (wrap in try catch and, in case of failure, return null or empty string or whatever works here).

Reply:
Can't do it in a seperate method. that stacktrace needs to be in the exact method that was called

Reply:
that's the reason WriteLine(string) and WriteLine(params object[]) are now more complicated

Reply:
and why there's now a private WriteLine(string senderName, string SenderFile, MessageType type, string line)

Reply:
fixed in latest commit